translations/ui-search/en.json

```diff
@@ -22,5 +22,39 @@
   "searchableIndexes.contributor": "Contributor",
   "searchableIndexes.subject": "Subject",
   "searchableIndexes.classification": "Classification",
-  "searchableIndexes.publisher": "Publisher"
+  "searchableIndexes.publisher": "Publisher",
+
+  "filters.sources.local": "Local",
+  "filters.sources.kb": "Knowledge Base",
+
+  "filters.resourceTypes.audio": "Audio",
+  "filters.resourceTypes.books": "Books",
+  "filters.resourceTypes.audiobooks": "Audiobooks",
+  "filters.resourceTypes.bookseries": "Bookseries",
+  "filters.resourceTypes.databases": "Databases",
+  "filters.resourceTypes.ebooks": "eBooks",
+  "filters.resourceTypes.kits": "Kits",
+  "filters.resourceTypes.maps": "Maps",
+  "filters.resourceTypes.music": "Music",
+  "filters.resourceTypes.newspapers": "Newspapers",
+  "filters.resourceTypes.newsletters": "Newsletters",
+  "filters.resourceTypes.periodicals": "Periodicals",
+  "filters.resourceTypes.posters": "Posters",
+  "filters.resourceTypes.reports": "Reports",
+  "filters.resourceTypes.proceedings": "Proceedings",
+  "filters.resourceTypes.thesisanddissertation": "Thesis and Dissertation",
+  "filters.resourceTypes.unspecified": "Unspecified",
+  "filters.resourceTypes.video": "Video",
+  "filters.resourceTypes.webresources": "Web Resources",
```

Comment:
Please update this phrase to [use sentence case](https://ux.folio.org/docs/guidelines/language-rules/text-case/).  There may be other examples of phrases in this `en.json` file that may need to be updated as well.